cmd/gitops-bucket-server/main.go

```diff
@@ -14,13 +14,24 @@ import (
 )
 
 func main() {
+	logger := log.New(os.Stdout, "", 0)
+
+	if os.Getenv("AWS_ACCESS_KEY_ID") == "" {
```

Comment:
I wonder if we should make these command-line flags just as the other parameters. This would bring all parameters into alignment.

Reply:
I would rather want to bind them via envs with Secret. It would be hard to hide those values if they were on the command line. wdyt?

Reply:
Yes, agreed.